v3.2/glfw/input.go

```diff
@@ -390,23 +390,29 @@ func (w *Window) SetCursorPos(xpos, ypos float64) {
 // Like all other coordinate systems in GLFW, the X-axis points to the right and the Y-axis points down.
 func CreateCursor(img image.Image, xhot, yhot int) *Cursor {
 	var img_c C.GLFWimage
-	var pixPointer *uint8
+	var pixels []uint8
 	b := img.Bounds()
 
 	switch img := img.(type) {
 	case *image.RGBA:
-		pixPointer = &img.Pix[0]
+		pixels = img.Pix
 	default:
 		m := image.NewRGBA(image.Rect(0, 0, b.Dx(), b.Dy()))
 		draw.Draw(m, m.Bounds(), img, b.Min, draw.Src)
-		pixPointer = &m.Pix[0]
+		pixels = m.Pix
 	}
 
+	pixPtr_c, pixFree_c := str(string(pixels) + "\x00")
```

Comment:
What is `str`? Shouldn't it be `strs`, the new func you've added? I'm guessing it's a typo, right?